ERCS/erc-7913.md

```diff
@@ -0,0 +1,107 @@
+---
+eip: 7913
+title: Key Verifiers
+description: Interface for the verification of signatures by address-less keys
+author: Hadrien Croubois (@Amxx), Ernesto García (@ernestognw), Francisco Giordano (@frangio), Aryeh Greenberg (@ar00)
+discussions-to: https://ethereum-magicians.org/t/erc-7913-key-verifiers/23262
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-03-21
+requires: 1271
```

Comment:
This shouldn't be needed. If we remove the backwards-compatible verification procedure from the specification it will be possible to remove it.